emperor/flat_plots.py

```diff
@@ -0,0 +1,346 @@
+from matplotlib import use
+use('Agg')  # noqa
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_3x3(ordination, offset, mapping, field, colors, null_kw=None,
```

Comment:
I think this request is for a fundamentally different driver method and is in my opinion out of scope for this PR. The layout is intentionally 3x3 and ridged as it ensures consistency between multiple figure sets such that flipping through say, Preview.app, allows for seeing relationships between different sets of metadata.

The color mapping functionality is required for the current use case with the EMP. I'd be happy to consider the case of `colors=None` and let matplotlib resolve the colors. 

Shapes are supported as you can pass in figure specific parameters with `focus_kw` and `null_kw` although they are not category value specific.